src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -67,9 +74,11 @@ public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFact
         public override void LogFunctionExecutionEvent(string executionId, string siteName, int concurrency, string functionName,
             string invocationId, string executionStage, long executionTimeSpan, bool success)
         {
-            var logger = _loggerFactory.GetOrCreate(FunctionsExecutionEventsCategory);
+            bool logBackoffEnabled = !_functionsHostingConfigOptions.Value?.DisableLinuxLogBackoff ?? true;
```

Comment:
_functionsHostingConfigOptions.Value is always should have value, can. be simplified to:

`bool logBackoffEnabled = !_functionsHostingConfigOptions.Value.DisableLinuxLogBackoff`

Reply:
Thank you for the review. Changed this